src/components/StatsBoxGrid.tsx

```diff
@@ -170,14 +55,23 @@ const GridItem: React.FC<IGridItemProps> = ({ metric, dir }) => {
   ) : isLoading ? (
     <StatLoadingMessage />
   ) : (
-    <StatRow>
-      <span>
+    <VStack>
+      <Box>
         {state.value}{" "}
         <Tooltip content={tooltipContent(metric)}>
-          <StyledIcon name="info" />
+          <Icon
+            as={MdInfoOutline}
+            width="24px"
+            height="24px"
```

Comment:
Nitpick: This can be simplified to the `boxSize` prop along with using the appropriate token value.
```suggestion
            boxSize={6}
```